src/registerDynamicSymbol.c

```diff
@@ -0,0 +1,27 @@
+// RegisteringDynamic Symbols
+
+#include <R.h>
+#include <Rinternals.h>
+#include <R_ext/Rdynload.h>
+#include "test.h"
+#include "EuclideanDistanceTransformation.h"
+
+static R_NativePrimitiveArgType myC_type[] = {
+   INTSXP,INTSXP,INTSXP,INTSXP,INTSXP,INTSXP
+};
+static const R_CMethodDef cMethods[] = {
+   {"test",  (DL_FUNC) &test, 0},
+   {"EuclideanDistanceTransformation", (DL_FUNC) &EuclideanDistanceTransformation, 6,myC_type},
```

Comment:
If we remove `EuclideanDistanceTransformation`, we should also remove the references to it here. We can always add them back once we get the libtopotoolbox integration working.